::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Fri, 14 Jul 2017 19:26:11 +0000
Subject: [PATCH] linux/system: AArch64: Remove mcelog

mcelog is not available on AArch64 hardware, so skip trying to install
it. This implies Ceilometer should not use mcelog on AArch64.

Signed-off-by: Guillermo Herrero <Guillermo.Herrero@enea.com>
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 linux/system/single.yml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/linux/system/single.yml b/linux/system/single.yml
index ef23a39..f07fa68 100644
--- a/linux/system/single.yml
+++ b/linux/system/single.yml
@@ -11,8 +11,6 @@ parameters:
           version: latest
         cloud-init:
           version: purged
-        mcelog:
-          version: latest
       kernel:
         modules:
           - nf_conntrack
